packages/devextreme-vue/README.md

```diff
@@ -1,24 +1,25 @@
-# DevExtreme Vue UI and Visualization Components #
+# DevExtreme for Vue
 
-[![Run Status](https://api.shippable.com/projects/5ab4c6354a24a207009ec636/badge?branch=master)](https://app.shippable.com/github/DevExpress/devextreme-vue)
-[![NPM](https://img.shields.io/npm/v/devextreme-vue.svg?maxAge=43200)](https://www.npmjs.com/package/devextreme-vue)
+[![NPM version](https://img.shields.io/npm/v/devextreme-vue.svg?maxAge=43200)](https://www.npmjs.com/package/devextreme-vue)
 
-This project allows you to use [DevExtreme components](http://js.devexpress.com/Demos/WidgetsGallery/) in [Vue](https://vuejs.org) applications.
+The `devextreme-vue` package contains the [DevExtreme component suite](http://js.devexpress.com/Demos/WidgetsGallery/) for the [Vue](https://vuejs.org) framework.
 
-* [Documentation](https://js.devexpress.com/Documentation/Guide/Vue_Components/DevExtreme_Vue_Components/)
-* [Technical Demos](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/Overview/Vue/Light/)
-* [API Reference](http://js.devexpress.com/Documentation/ApiReference/)
-* [DevExtreme Vue Application Template](https://github.com/DevExpress/devextreme-vue-template)
-* [Live example](https://codesandbox.io/s/github/lukyanovas/devextreme-vue-example)
+* [Documentation](https://js.devexpress.com/Vue/Documentation/Guide/Vue_Components/DevExtreme_Vue_Components/)
+* [Component Demos](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/Overview/Vue/Light/)
+* **Template Gallery**
+    * [Fluent theme](https://devexpress.github.io/devextreme-ui-template-gallery/fluent.blue.light/vue/default/#/crm-contact-list)
+    * [Generic theme](https://devexpress.github.io/devextreme-ui-template-gallery/generic.light.compact/vue/default/#/crm-contact-list)
+* [Vue Application Template](https://github.com/DevExpress/devextreme-vue-template)
```

Comment:
the link should be https://js.devexpress.com/Vue/Documentation/Guide/Vue_Components/Application_Template/